src/internal/modules/charge-information/forms/use-abstraction-data.js

```diff
@@ -35,10 +53,12 @@ const useAbstractionDataForm = request => {
   return f;
 };
 
-const useAbstractionDataSchema = () => {
+const useAbstractionDataSchema = (request) => {
+  const { chargeVersions } = request.pre;
+  const validIds = filterChargeVersions(chargeVersions).map(cv => (cv.status === 'current') ? cv.id : null);
```

Comment:
I think the filter already ensures there are only current ones - could this line be simplified to:
```
const validIds = filterChargeVersions(chargeVersions).map(cv => cv.id);
```